CHANGELOG.md

```diff
@@ -57,7 +57,7 @@ Release date: 2019-10-22
 > **NOTE**: To revert to previous release behavior, pass the system property
 > `org.jenkinsci.plugins.durabletask.BourneShellScript.FORCE_SHELL_WRAPPER=true` to the Java command line used to start Jenkins.
 
--   Update ssh-slaves ([PR \#100](https://github.com/jenkinsci/durable-task-plugin/pull/100))
+-   Update ssh-agents ([PR \#100](https://github.com/jenkinsci/durable-task-plugin/pull/100))
```

Comment:
Slightly different for linguistic reasons.  The previous entry used the "plugin ID" which is not well known to users and is unchanging even for inclusive naming.  Let's use the plugin name rather than the identifier
```suggestion
-   Update SSH Build Agents plugin ([PR \#100](https://github.com/jenkinsci/durable-task-plugin/pull/100))
```